### ISSUE TITLE
Empty Project

### ISSUE THREAD

**Issue by zirinisp** [choncou/DarkSkyWeatherApi#1]
Am I missing something or this is an empty project?

**Comment by choncou** [comment]
@zirinisp Yes the project is empty.. I planned to work on it a while ago but never got the chance to.
